src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+import { MainPage } from "./pages/MainPage";
+import { ProfilePage } from "./pages/ProfilePage";
+import { Navbar } from "./components/Navbar";
```

Comment:
It looks like you renamed the folder to 'Components' so now you need to update imports as well.
```suggestion
import { Navbar } from "./Components/Navbar";
```